### PR TITLE
Add api tests for addresses

### PR DESCRIPTION
**Pull request by asbiin**

### CODE REVIEW

File: tests/Api/ApiAdressesTest.php [hunk 1]
```diff
@@ -131,4 +130,115 @@ public function test_address_get_addressid()
...
+            'contact_id' => $contact->id,
+            'name' => 'address name',
+            'street' => 'street',
+            'postal_code' => '12345',
+            'country' => 'FR'
+        ]);
+
+        $response->assertJsonFragment([
+            'object' => 'address',
+            'id' => 1,
```
Comment (djaiss) [inline tests/Api/ApiAdressesTest.php]: IDs shouldn't be hardcoded. In my locale environment, each time I run the tests, the ID increments, making this test fail everytime.

Comment (djaiss) [reply tests/Api/ApiAdressesTest.php]: @asbiin ping.

Comment (djaiss) [reply tests/Api/ApiAdressesTest.php]: Here you can't really test the ID number. You can test the presence of an ID though, or you could test that the table `addresses` contains no row, then one row after your update is complete.

Comment (asbiin) [reply tests/Api/ApiAdressesTest.php]: Good idea

Comment (asbiin) [reply tests/Api/ApiAdressesTest.php]: Fixed by #1333

**Pull request merged**